sdk/keyvault/azure-keyvault-administration/azure/keyvault/administration/_access_control_client.py

```diff
@@ -0,0 +1,112 @@
+
+    # pylint:disable=protected-access
+
+    @distributed_trace
+    def create_role_assignment(self, role_scope, role_assignment_name, role_definition_id, principal_id, **kwargs):
+        # type: (str, str, str, str, **Any) -> RoleAssignment
+        """Create a role assignment.
+
+        :param str role_scope: scope the role assignment will apply over
+        :param role_assignment_name: a name for the role assignment. Must be a UUID.
+        :type role_assignment_name: str or uuid.UUID
+        :param str role_definition_id: ID of the role's definition
+        :param str principal_id: ID of the principal which will be assigned the role. This maps to the ID inside the
+            Active Directory. It can point to a user, service principal, or security group.
+        :rtype: RoleAssignment
+        """
+        create_parameters = self._client.role_assignments.models.RoleAssignmentCreateParameters(
+            properties=self._client.role_assignments.models.RoleAssignmentProperties(
+                principal_id=principal_id, role_definition_id=str(role_definition_id)
+            )
+        )
+        assignment = self._client.role_assignments.create(
+            vault_base_url=self._vault_url,
+            scope=role_scope,
+            role_assignment_name=role_assignment_name,
+            parameters=create_parameters,
+            **kwargs
+        )
+        return RoleAssignment._from_generated(assignment)
+
+    @distributed_trace
+    def delete_role_assignment(self, role_scope, role_assignment_name, **kwargs):
+        # type: (str, str, **Any) -> RoleAssignment
+        """Delete a role assignment.
+
+        :param str role_scope: the assignment's scope
+        :param role_assignment_name: the assignment's name. Must be a UUID.
+        :type role_assignment_name: str or uuid.UUID
+        :returns: the deleted assignment
+        :rtype: RoleAssignment
+        """
+        assignment = self._client.role_assignments.delete(
+            vault_base_url=self._vault_url, scope=role_scope, role_assignment_name=str(role_assignment_name), **kwargs
+        )
+        return RoleAssignment._from_generated(assignment)
+
+    @distributed_trace
+    def get_role_assignment(self, role_scope, role_assignment_name, **kwargs):
+        # type: (str, str, **Any) -> RoleAssignment
+        """Get a role assignment.
+
+        :param str role_scope: the assignment's scope
+        :param role_assignment_name: the assignment's name. Must be a UUID.
+        :type role_assignment_name: str or uuid.UUID
+        :rtype: RoleAssignment
+        """
+        assignment = self._client.role_assignments.get(
+            vault_base_url=self._vault_url, scope=role_scope, role_assignment_name=str(role_assignment_name), **kwargs
+        )
+        return RoleAssignment._from_generated(assignment)
+
+    @distributed_trace
+    def list_role_assignments(self, role_scope, **kwargs):
+        # type: (str, **Any) -> Iterable[RoleAssignment]
```

Comment:
I think typing should be `ItemPaged` instead of `Iterable`. This should also be fixed in the other libraries too, I've opened an issue for it [here](https://github.com/Azure/azure-sdk-for-python/issues/13373)